Vagrantfile

```diff
@@ -122,4 +119,17 @@ Vagrant.configure(2) do |config|
     provisioner.add_host '192.168.51.102', ['puppetagent02', 'puppetagent02.local']
   end
 
+  config.vm.provision :host_shell do |host_shell|
```

Comment:
These 3 blocks can be deleted. I add instructions on read.me file on these instead. Form past experience these 3 blocks have always been a bit hit and miss. anyway.